msbot/constants.py

```diff
@@ -19,19 +19,24 @@
 CHALLENGE = 'hub.challenge'
 SUBSCRIBE = 'subscribe'
 
+# Commands
 HELLO = 'hello'
 GOODBYE = 'goodbye'
 SEND = 'send'
+RECENT = 'recent'
 
-RESP_SUBBED = 'You are now subscribed. Say "goodbye" at any time to unsubscribe'
+# Responses
+RESP_SUBBED = "You are now subscribed. Say 'goodbye' at any time to unsubscribe"
 RESP_UNSUBBED = 'You have been unsubscribed from MythicSpoilerBot'
 RESP_ALREADY_SUBBED = 'You are already subscribed'
 RESP_ALREADY_UNSUBBED = 'You are not subscribed'
-RESP_INVALID_UNSUBBED = 'Invalid command. Say "hello" at any time to subscribe'
-RESP_INVALID_SUBBED = 'Invalid command. Say "goodbye" at any time to unsubscribe'
+RESP_INVALID_UNSUBBED = "Invalid command. Say 'hello' at any time to subscribe"
+RESP_INVALID_SUBBED = "Invalid command. Say 'recent' to get the latest batch of spoilers and say 'goodbye' at any time to unsubscribe"
```

Comment:
not necessary now, but in the future it might make more sense to just always have a `Invalid command. Say 'help' to see available commands`.

Reply:
Agree - would you want that message as part of this pull request?

Reply:
naw generally 1 pull request per feature